kubeconfig/src/Kubernetes/KubeConfig.hs

```diff
@@ -48,6 +48,7 @@ data Config = Config
   , authInfos      :: [NamedEntity AuthInfo "user"]
   , contexts       :: [NamedEntity Context "context"]
   , currentContext :: Text
+  , spec           :: Spec
```

Comment:
This should not be here. This is the type of a kubeconfig, whose format is defined here: https://github.com/kubernetes/client-go/blob/master/tools/clientcmd/api/v1/types.go